pyo3-ffi/src/cpython/unicodeobject.rs

```diff
@@ -30,6 +30,186 @@ use std::os::raw::{c_char, c_int, c_uint, c_void};
 // skipped Py_UNICODE_HIGH_SURROGATE
 // skipped Py_UNICODE_LOW_SURROGATE
 
+// generated by bindgen v0.63.0 (with small adaptations)
+#[repr(C)]
+#[derive(Copy, Clone, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd)]
```

Comment:
Coverage is suggesting some of these are unused, perhaps we can remove some (or maybe all) of them?

Reply:
You're right, I think none of them are needed.